src/inmanta/data/__init__.py

```diff
@@ -3992,14 +3992,18 @@ def _get_resource_logs_base_query(
         resource_id: m.ResourceIdStr,
         offset: int,
     ) -> Tuple[str, List[object]]:
+        # The query uses a like query to match resource id with a resource_version_id. This means we need to escape the % and _
+        # characters in the query
+        resource_id = resource_id.replace("%", "#%").replace("_", "#_") + "%"
```

Comment:
We probably have to escape itself # as well

Reply:
Might be good to test with a resource with a sequence of escaped chars

Reply:
I added an escape for the escape. I did a test on a database and that was indeed required